Jenkinsfile

```diff
@@ -3,11 +3,29 @@ pipeline {
     options {
         timestamps()
     }
+    environment {
+        CONTAINER_ENGINE_CMD=''
+        TEST_CONTAINER_NAME=''
+        TEARDOWN_RAN=0
+        GO_TOOLSET_IMAGE='registry.access.redhat.com/ubi9/go-toolset:1.18.9'
+    }
+
     stages {
-        stage('no-op') {
+        stage('Run Unit Tests') {
             steps {
-                sh "echo 'Jenkinsfile placeholder'"
+                sh '''
+                    . ./ci/helpers.sh
+                    . ./ci/unit_tests.sh
+                '''
             }
         }
     }
+
+    post { 
+        always {
+            sh 'teardown'
```

Comment:
this may fail if it's not in the context (i.e. if `ci/helpers.sh` is not sourced (not sure how environment is preserved in `post`